### DISCUSSION TITLE
Export Chat Option like Whatsapp

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#138] [Ideas & Feature Suggestions]
As you know in whatsapp we can export a chat to telegram or any other messaging apps. But it is NOT required for snapchat. 
I've messages saved from more than years ago, so if you can bring a feature to export a chat to .txt format then it will be a cool feature as there is no search bar to search for text messages in snapchat. So we can export the chats for sharing or storing or whatever.

And better, in addition, if you add a switch which will ask the date for the chats to be exported then it would be nice too. It's a quite amazing feature. Hope to see this soon from you.

Thank you! And as always, Good job for this project

**Comment by NMBlanco** [comment]
To add something, I think it would be cool to have an option that allows you to "Jump to first message" in chat. If this is somehow easy to do @rodit

**Reply by RevealedSoulEven** [reply]
Yeaaa that's cool too. But if you need to export a chat then it will be better to export it in .txt format or any other if he likes

**Comment by rodit** [comment]
Added in 1.7.2

**Reply by RevealedSoulEven** [reply]
Working great ❤️ thanks rodit
